src/desktop_release.js

```diff
@@ -298,6 +298,15 @@ class DesktopReleaseBuilder {
             const latestPath = path.join(this.appPubDir, 'update', 'macos', 'latest');
             logger.info('Write ' + buildVersion + ' -> ' + latestPath);
             await fsProm.writeFile(latestPath, buildVersion);
+
+            const pkg = JSON.parse(await fsProm.readFile(path.join(repoDir, 'package.json')));
+            const newInstallPath = path.join(`${pkg.productName}-${buildVersion}.dmg`);
```

Comment:
The `getMatchingFilesInDir` above will find the dmg so you could also take the filename from that which may be more reliable than generating it. Either probably fine though.

Reply:
Ah yeah, seems better, have changed to that.